pmctl/main.go

```diff
@@ -61,12 +61,12 @@ func init() {
 	// Let cobra ignore if we are running as "GUI" or not
 	cobra.MousetrapHelpText = ""
 
-	rootCmd.PersistentFlags().StringVar(&dataDir, "data", "", "set data directory")
-	rootCmd.PersistentFlags().StringVar(&databaseDir, "db", "", "alias to --data (deprecated)")
+	rootCmd.PersistentFlags().StringVar(&dataDir, "data", "", "Configures the data directory. It's also possible to configure the PORTMASTER_DATA environment variable.")
```

Comment:
Description somewhat unclear, would recommend:
`Configures the data directory. Alternatively, this can also be set via the environment variable PORTMASTER_DATA.